bedrock/etc/templates/etc/firefox/retention/thank-you-a.html

```diff
@@ -25,41 +25,45 @@
   <canvas id="confetti-canvas"></canvas>
   <header>
     <h1>
-      <span class="white">It's all</span></br>
+    {% if LANG.startswith('en') %}
+      <span class="white">It’s all</span>
       thanks to you
+    {% else %}
+      It’s all thanks to you
+    {% endif %}
     </h1>
     <p>
-      Choosing Firefox helps Mozilla make the Internet<br/>a better place. Here's what you can do next:
+      {{ _('Choosing Firefox helps Mozilla make the Internet a better place. Here’s what you can do next:') }}
     </p>
   </header>
   <section>
     <article>
-      <h1>Make Firefox your default browser</h1>
-      <div class="time">1 min action</div>
-        <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">Set as your default</a>
+      <h1>{{ _('Make Firefox your default browser') }}</h1>
+      <div class="time">{{ _('1 min action') }}</div>
```

Comment:
@peiying2 or @flodolo: Do these strings depicting time need any special treatment? These strings are meant to communicate the time ("1 min" or "4 min") necessary to complete the task ("action" or "read"). Would an L10n comment be helpful here?